rt/rest2.py

```diff
@@ -649,6 +651,10 @@ def edit_ticket(self, ticket_id: typing.Union[str, int], **kwargs: typing.Any) -
         if not msg:
             return True
 
+        for k in msg:
+            if re.search(REGEX_PATTERNS['does_not_exist'], k):
```

Comment:
```suggestion
            if REGEX_PATTERNS['does_not_exist'].search(k):
```

Reply:
:facepalm: ... thnx for catching that :+1:

Reply:
Sure that's also why I didn't notice it when testing, but it should not be called like that, looks ugly :-)